etc/PackageConfigurations/eos_package_description.json

```diff
@@ -15,9 +15,9 @@
                              
         {"src": "Assets/Plugins/*",                                "dest": "Runtime/"                             },
         {"src": "Assets/Plugins/Source/Core.meta",                 "dest": "Runtime/"                             },
-        {"src": "Assets/Plugins/Source/Core/*",                    "dest": "Runtime/Core/"                        },
-        {"src": "Assets/Plugins/Source/Core/Utility/*",            "dest": "Runtime/Core/Utility/"                },
-        {"src": "Assets/Plugins/Source/Core/Utility/Extensions/*", "dest": "Runtime/Core/Utility/Extensions/"     },
+        {"src": "com.playeveryware.com/Runtime/Core/*",                    "dest": "Runtime/Core/"                        },
```

Comment:
Above this, it is clear that we're maintaining "Assets/Plugins/Source/Core.meta". Should there be an associated .meta file for "com.playeveryware.com/Runtime/Core/", or is this outside of the folder structure requiring meta files?

Reply:
Nice catch! This does need to be fixed.